src/components/Home.jsx

```diff
@@ -185,17 +189,19 @@ const Home = ({ currentLanguage, dispatch, history, language, rtl, translate })
       <div className="home-page__partners">
         <h2>{translate('home.institutions')}</h2>
         <div>
-          <ZooniverseLogotype />
+          <a href="https://www.zooniverse.org/" target="_blank" rel="noopener noreferrer"><ZooniverseLogotype /></a>
 
-          <img alt="Penn" src={ImgPenn} />
-          <img alt="The Jewish Theological Seminary" src={ImgJTS} />
-          <img alt="Princeton Geniza Project" src={ImgPrinceton} />
+          <a href="http://www.library.upenn.edu/" target="_blank" rel="noopener noreferrer"><img alt="Penn" src={ImgPenn} /></a>
+          <a href="http://www.jtsa.edu/" target="_blank" rel="noopener noreferrer"><img alt="The Jewish Theological Seminary" src={ImgJTS} /></a>
+          <a href="https://www.princeton.edu/~geniza/" target="_blank" rel="noopener noreferrer"><img alt="Princeton Geniza Project" src={ImgPrinceton} /></a>
 
-          <img alt="Cambridge" src={ImgCambridge} />
-          <img alt="Manchester" src={ImgManchester} />
-          <img alt="Bodleian" src={ImgBodleian} />
-          <img alt="The e-Lijah Lab at the University of Haifa" src={ImgElijah} />
-          <img alt="The Centre for Interdisciplinary Research of the Cairo Genizah at the University of Haifa" src={GenizahCentre} />
+          <a href="http://www.lib.cam.ac.uk/collections/departments/taylor-schechter-genizah-research-unit" target="_blank" rel="noopener noreferrer"><img alt="Cambridge" src={ImgCambridge} /></a>
+          <a href="http://www.library.manchester.ac.uk/rylands/" target="_blank" rel="noopener noreferrer"><img alt="Manchester" src={ImgManchester} /></a>
+          <a href="https://www.bodleian.ox.ac.uk/" target="_blank" rel="noopener noreferrer"><img alt="Bodleian" src={ImgBodleian} /></a>
+          <a href="#" target="_blank" rel="noopener noreferrer"><img alt="The e-Lijah Lab at the University of Haifa" src={ImgElijah} /></a>
```

Comment:
Hmm - on second thoughts, instead of this placeholder, I'm going to strip the <a> here first _until_ we hear back on whether or not e-Lijah has a website, then I'll pop a quick future PR to add in the URL. I don't want to hold up this PR merge with waiting for a website URL.